rtt/ServiceRequester.cpp

```diff
@@ -67,7 +67,7 @@ namespace RTT
             log(Error) << "OperationCaller with name '" + isb.getName() + "' already present." << endlog();
             return false;
         }
-        mmethods.insert(make_pair<std::string, OperationCallerBaseInvoker*> (isb.getName(), &isb));
+        mmethods.insert(make_pair<std::string, OperationCallerBaseInvoker*> (std::string(isb.getName()), &isb));
```

Comment:
Please change this line to
```cpp
mmethods.insert(pair<std::string, OperationCallerBaseInvoker*> (isb.getName(), &isb));
```
or
```cpp
mmethods.insert(make_pair(isb.getName(), &isb));
```
Both variants compile with and without C++11.

The whole point of `std::make_pair` is to construct pairs without having to specify the types explicitly.
See http://stackoverflow.com/questions/14623958/breaking-change-in-c11-with-make-pair-ty1-val1-const-ty2-val2.